tests/integration_tests/cachekeys/api_tests.py

```diff
@@ -165,3 +176,40 @@ def test_invalidate_existing_caches(invalidate):
         .datasource_uid
         == "X__table"
     )
+
+
+class TestWarmUpCache(SupersetTestCase):
+    @pytest.mark.usefixtures(
+        "load_energy_table_with_slice", "load_birth_names_dashboard_with_slices"
+    )
+    def test_warm_up_cache(self):
```

Comment:
can you add test cases for chart_id, dashboard_id, table_name not found and other possible failures at the API level.
Would be great to add unittests at the command level

Reply:
added tests for API failures and some basic command tests